test/hip-583/HIP583.js

```diff
@@ -242,8 +230,9 @@ describe('HIP583 Test Suite', function () {
           //sending some HBARs, so the hollow account have some to cover the transaction
           await signers[0].sendTransaction({
             to: hollowWalletAddress,
-            value: ethers.utils.parseEther('1'),
+            value: ethers.utils.parseEther('20'),
```

Comment:
Q: why the bump to 20 HBARs. Is this an accurate value or an estimate far above what's needed?
This will contribute to the cost of the tests

Reply:
That value was determined by @quiet-node  to allow the test to pass at the time.  I was able to reduce it to 14.

Reply:
Sure